steam/client/builtins/friends.py

```diff
@@ -156,7 +156,7 @@ def add(self, steamid_or_accountname_or_email):
         """
         m = MsgProto(EMsg.ClientAddFriend)
 
-        if isinstance(steamid_or_accountname_or_email, (intBase, int)):
+        if isinstance(steamid_or_accountname_or_email, (int, int)):
```

Comment:
now that you removed `intBase`, you can replace `(int, int)` with just `int`

https://docs.python.org/3/library/functions.html#isinstance
> isinstance(object, classinfo)
> [...]
> If classinfo is a tuple of type objects (or recursively, other such tuples) or a [Union Type](https://docs.python.org/3/library/stdtypes.html#types-union) of multiple types, return True if object is an instance of any of the types

Reply:
updated!